scripts/deploy.js

```diff
@@ -5,10 +5,12 @@ async function main() {
     console.log("Account balance:", (await deployer.getBalance()).toString());
 
     const BnomialNFTContract = await ethers.getContractFactory("BnomialNFT");
+    const NFTcontract = await BnomialNFTContract.deploy();
+    console.log("NFTcontract Contract address:", NFTcontract.address);
 
-    const contract = await BnomialNFTContract.deploy();
-
-    console.log("Contract address:", contract.address);
+    const BinomialToken1Contract = await ethers.getContractFactory("BinomialToken1");
```

Comment:
We should have separate scripts to deploy the NFT token and the ERC-20 token.

Reply:
Question. Do you want to call first script deployNFT.js and the other script deployToken.js? @haltakov

Reply:
I will also add the new script to package.json

Reply:
Yeah, deployNFT and deployToken sounds good!